src/objects/RecurringDonationChangeHistory__c.object

```diff
@@ -0,0 +1,389 @@
+        <label>Change Type</label>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <valueSetDefinition>
+                <sorted>false</sorted>
+                <value>
+                    <fullName>Upgrade</fullName>
+                    <default>false</default>
+                    <label>Upgrade</label>
+                </value>
+                <value>
+                    <fullName>Downgrade</fullName>
+                    <default>false</default>
+                    <label>Downgrade</label>
+                </value>
+            </valueSetDefinition>
+        </valueSet>
+    </fields>
+    <fields>
+        <fullName>Effective_Date__c</fullName>
+        <description>The Effective Date of the Recurring Donation when this change occurred.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Effective Date of the Recurring Donation when this change occurred.</inlineHelpText>
+        <label>Effective Date</label>
+        <required>true</required>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>New_Amount__c</fullName>
+        <description>The new Amount for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The new Amount for the related Recurring Donation.</inlineHelpText>
+        <label>New Amount</label>
+        <precision>18</precision>
+        <required>false</required>
+        <scale>2</scale>
+        <trackTrending>false</trackTrending>
+        <type>Currency</type>
+    </fields>
+    <fields>
+        <fullName>New_Campaign__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The new Campaign for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The new Campaign for the related Recurring Donation.</inlineHelpText>
+        <label>New Campaign</label>
+        <referenceTo>Campaign</referenceTo>
+        <relationshipLabel>Recurring Donation Change History (New Campaign)</relationshipLabel>
+        <relationshipName>RD_Change_History_New_Campaign</relationshipName>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>New_Payment_Method__c</fullName>
+        <description>The new Payment Method for the related Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The new Payment Method for the related Recurring Donation.</inlineHelpText>
+        <label>New Payment Method</label>
+        <length>255</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>New_Schedule__c</fullName>
```

Comment:
This prevent customers from reporting on changes from Monthly to Yearly or every 1 month to every 3 months, etc. Mainly I'm thinking about the report aspect of this. As a text field that concatenates the schedule into a single string, it won't be parseable for reports or queries.

Reply:
We've been back and forth on this a lot, I'm fine separating them at this point due to the confusion and questions the concatenated string has caused.